pkg/cli/root.go

```diff
@@ -0,0 +1,47 @@
+package cli
```

Comment:
the file name here is a bit horrible? should better use the real cmd name `ocm-agent.go` IMO.

Reply:
Well, I tried following convention as per https://cobra.dev/#create-rootcmd. `main()` would execute the CLI's command and the root command would have the mapping of all the subcommands. I did come across different conventions from different repos though. Another example is https://github.com/openshift/osd-network-verifier/blob/main/cmd/root.go 

Let me know if this is ok. Else, we can change it too.